import java.util.Scanner;

class Exception {
  public static void main(String[] args) {
	  
try{	
 
  int a, b, result = 0;
 
  Scanner input = new Scanner(System.in);
  System.out.println("Input two integers");

  
  //catch using Arithmetic Exception
  a=input.nextInt();
  b=input.nextInt();
  result = a/b;
  
  while ( result!= -1)
	{
		System.out.println("Result =" + result);
	}
  }catch (ArithmeticException s)
  {
	  
  } catch (NumberFormatException e) {
	  


  }
  }
}


